tests/new_relay/run.sh

```diff
@@ -39,11 +40,26 @@ function run() {
 		"\"worker\": \"worker1\"" 1 \
 		"\"worker\": \"worker2\"" 1
 
+	# worker1 and worker2 has one realy job and worker3 have none.
+	check_metric $WORKER1_PORT "dm_relay_binlog_file{node=\"relay\"}" 3 0 2
+	check_metric $WORKER1_PORT "dm_relay_exit_with_error_count" 3 -1 1
+	check_metric $WORKER2_PORT "dm_relay_binlog_file{node=\"relay\"}" 3 0 2
+	check_metric $WORKER2_PORT "dm_relay_exit_with_error_count" 3 -1 1
+	check_metric_not_contains $WORKER3_PORT "dm_relay_binlog_file" 3
+
 	dmctl_start_task_standalone $cur/conf/dm-task.yaml "--remove-meta"
 	check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
 
 	run_sql_file $cur/data/db1.increment.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1
 
+	# relay task tranfer to worker1 with no error.
+	check_metric $WORKER1_PORT "dm_relay_data_corruption" 3 -1 1
+	check_metric $WORKER1_PORT "dm_relay_read_error_count" 3 -1 1
+	check_metric $WORKER1_PORT "dm_relay_write_error_count" 3 -1 1
+	# check worker relay space great than 0 9223372036854775807 is 2**63 -1
+	sleep 5
```

Comment:
is this needed?

Reply:
need  in ci test, but i don't know why,maybe another unstable test >.<

Reply:
https://github.com/pingcap/dm/blob/48dacb1aed28184a325b27a41e3e9f06ae808045/relay/metrics.go#L153-L177

could add a failpoint to reduce ticker interval

Reply:
added, fialpoint is so magic !!